src/Commands/AliasesCommand.php

```diff
@@ -27,7 +27,7 @@ class AliasesCommand extends TerminusCommand implements SiteAwareInterface
      * @authorize
      *
      * @command aliases
-     * @aliases drush:aliases
+     * @aliases drush:aliases,alpha:aliases
```

Comment:
We've been separating the aliases with spaces.
https://github.com/pantheon-systems/terminus/blob/08e8264b29717e6fc716414ec6fa15cf035027bc/src/Commands/MachineToken/ListCommand.php#L24

Reply:
Swap the comma for a space and it's perfect